inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+public class Restaurant {
+    int numberOfStars;
```

Comment:
These variables should have class modifiers (ie: public, private, or protected).